core/validatorapi/eth2types.go

```diff
@@ -167,3 +167,13 @@ type syncCommitteeDutiesResponse struct {
 type syncCommitteeContributionResponse struct {
 	Data *altair.SyncCommitteeContribution `json:"data"`
 }
+
+// nodeVersionResponse defines the response to the node version endpoint.
+// See: https://ethereum.github.io/beacon-APIs/#/Node/getNodeVersion
+type nodeVersionResponse struct {
+	Data nodeVersionInner `json:"data"`
+}
+
+type nodeVersionInner struct {
```

Comment:
Is this struct needed? Can't we merge it into `nodeVersionResponse`?

The struct is small anyway, and only used to return one value.

Reply:
+1
Something like:
```
type nodeVersionResponse struct {
	Data struct {
		Version string `json:"version"`
	} `json:"data"`
}
```